js/src/toast.js

```diff
@@ -96,14 +94,24 @@ class Toast {
       return
     }
 
-    if (this._config.animation) {
-      this._element.classList.add(ClassName.FADE)
+    if (this._config.transitionName) {
+      this._element.classList.add(`${this._config.transitionName}-enter`)
```

Comment:
You can pass multiple classes IIRC.

Reply:
According to https://caniuse.com/#feat=classlist it seems not 😟